src/consumer/__tests__/runner.spec.js

```diff
@@ -247,22 +247,21 @@ describe('Consumer > Runner', () => {
     beforeEach(async () => {
       offsets = { topics: [{ topic: topicName, partitions: [{ offset: '1', partition }] }] }
       await runner.start()
-    })
 
-    it('should commit offsets while running', async () => {
-      // Clear state
+      consumerGroup.join.mockClear()
       consumerGroup.commitOffsetsIfNecessary.mockClear()
       consumerGroup.commitOffsets.mockClear()
+    })
 
+    it('should commit offsets while running', async () => {
       await runner.commitOffsets(offsets)
+
       expect(consumerGroup.commitOffsetsIfNecessary).toHaveBeenCalledTimes(0)
-      expect(consumerGroup.commitOffsets).toHaveBeenCalledTimes(1)
+      expect(consumerGroup.commitOffsets.mock.calls.length).toBeGreaterThanOrEqual(1)
```

Comment:
Sometimes the `eachMessage` has time to run, so because of `autoCommit` `commitOffsets` has been called twice by the time we assert.